make/uncrustify.cfg

```diff
@@ -0,0 +1,147 @@
+# run with FLIGHT=some_file.c make uncrustify_flight
+
```

Comment:
There's a program called 'universalindentgui' that you can load this into and live indent a source file.